.github/workflows/Pipeline.yml

```diff
@@ -114,7 +112,7 @@ jobs:
     uses: pyTooling/Actions/.github/workflows/ArtifactCleanUp.yml@r0
     needs:
       - Params
-      - UnitTesting
+      - PublishTestResults
```

Comment:
When we discussed needs rules, we agreed on using:
* the artifact creating job, and
* the last job using an artifact.

So we would need `PublishTestResults` in addition to `UnitTesting`.

Reply:
Done.